src/user-agent/browser/edge.ts

```diff
@@ -0,0 +1,11 @@
+import {Browser, TOffset} from "./base";
```

Comment:
Have you checked out the yano `is` class?  A majority of these user-agent tests are covered there....is that something that would work for you?

Reply:
Yes, removed user-agent.